docs/advanced/subscriptions.md

```diff
@@ -40,74 +40,64 @@ When we define this function it must return an "Observable-like" object, which n
+import { createClient as createWSClient } from 'graphql-ws';
 
-const subscriptionClient = new SubscriptionClient('ws://localhost/graphql', { reconnect: true });
+const wsClient = createWSClient({
+  url: 'ws://localhost/graphql',
+});
 
-const client = new Client({
+const client = createClient({
   url: '/graphql',
   exchanges: [
     ...defaultExchanges,
     subscriptionExchange({
-      forwardSubscription(operation) {
-        return subscriptionClient.request(operation);
-      },
+      forwardSubscription: (operation) => ({
+        subscribe: (sink) => ({
+          unsubscribe: wsClient.subscribe(operation, sink),
+        }),
+      }),
     }),
   ],
 });
 ```
 
-In this example, we're creating a `SubscriptionClient`, are passing in a URL and some parameters,
-and are using the `SubscriptionClient`'s `request` method to create a Subscription Observable, which
-we return to the `subscriptionExchange` inside `forwardSubscription`.
+[Read more on the `graphql-ws` README.](https://github.com/enisdenjo/graphql-ws/blob/master/README.md)
 
-[Read more about `subscription-transport-ws` on its README.](https://github.com/apollographql/subscriptions-transport-ws/blob/master/README.md)
+## Setting up `subscriptions-transport-ws`
 
-## Setting up `graphql-ws`
+For backends supporting `subscriptions-transport-ws`, [Apollo's `subscriptions-transport-ws`
+package](https://github.com/apollographql/subscriptions-transport-ws) can be used.
 
-If your GraphQL API server is using [graphql-ws](https://github.com/enisdenjo/graphql-ws),
-you'll be able to use it here too!
+> Note: `subscriptions-transport-ws` is deprecated, consider using `graphql-ws` (above)
 
 ```js
-import { createClient, defaultExchanges, subscriptionExchange } from 'urql';
-import { createClient as createWSClient } from 'graphql-ws';
+import { Client, defaultExchanges, subscriptionExchange } from 'urql';
+import { SubscriptionClient } from 'subscriptions-transport-ws';
 
-const wsClient = createWSClient({
-  url: 'ws://localhost/graphql',
-});
+const subscriptionClient = new SubscriptionClient('ws://localhost/graphql', { reconnect: true });
 
-const client = createClient({
+const client = new Client({
   url: '/graphql',
   exchanges: [
     ...defaultExchanges,
     subscriptionExchange({
-      forwardSubscription(operation) {
-        return {
-          subscribe: sink => {
-            const dispose = wsClient.subscribe(operation, sink);
-            return {
-              unsubscribe: dispose,
-            };
-          },
-        };
-      },
+      forwardSubscription: (operation) => subscriptionClient.request(operation)
     }),
   ],
 });
 ```
 
-We create a WebSocket client with the necessary options and use the `subscribe` method from it to
-create a Subscription Observable, which we return to the `subscriptionExchange` inside `forwardSubscription`.
+In this example, we're creating a `SubscriptionClient`, are passing in a URL and some parameters,
```

Comment:
I think it's inconsistent to keep this and delete the other description. I'm actually not sure whether to keep both or delete both, but better to be consistent 🤷 I'll leave it up to you 😸

Reply:
Sorry yeah that was a whimsical change - I'll add back in the old one for now to keep this scoped :+1: